src/app/GalleryManager.jsx

```diff
@@ -84,17 +84,18 @@ export default class GalleryManager extends Component {
     return (
       <div>
         <div style={styles.galleryTitle}>
-          <TextField
-            value={gallery.headline}
-            onChange={this.setHeadline.bind(this)}
+          <input
             style={styles.galleryTitles}
-            label="Write a headline (optional)" />
+            type="text"
+            value={gallery.headline}
+            placeholder="Headline"
+            onChange={this.setHeadline.bind(this)} />
```

Comment:
oof. We should use this architecture throughout the app. I'm sure this is part of our performance woes. thanks @okbel